app/web/src/components/AssetEditorTabs.vue

```diff
@@ -94,43 +93,41 @@ watch(
       // updating tab list
       currentTabs.value = tabs;
     }
+
+    if (funcStore.selectedFuncId)
+      tabGroupRef.value?.selectTab(funcStore.selectedFuncId);
+
+    // still dont know what is racing and removing the querystring from the URL
```

Comment:
@wendybujalski this is what I've done for the moment...<img src="https://media1.giphy.com/media/853jNve3ljqrYrcSOK/giphy.gif"/>

Reply:
![spin-shrug](https://github.com/systeminit/si/assets/8495209/cacae6cb-3f4e-42c1-94b7-e2be27867143)